### PR TITLE
Updating more code to python3

### PR DESCRIPTION
**Pull request by kaelingre**
Some more changes to make the import functions work with python3.

### CODE REVIEW

File: fuchsia.py [hunk 1]
```diff
@@ -1794,15 +1794,15 @@ def import_matrix_mathematica(f):
     from a file-like object.
     """
     data =  f.read()
-    data = data.translate(None, ' \n')
+    data = data.translate(str.maketrans('','',' \n'))
```
Comment (magv) [inline fuchsia.py:1797]: This doesn't work on Python2, doesn't it?

Comment (kaelingre) [reply fuchsia.py:1797]: Not sure to be honest. Note that most of the code has already been updated to python3 in the latest commit. Maybe branch for python2 vs python3?

Comment (magv) [reply fuchsia.py:1797]: Nah, no branches needed so far, I think `data.replace(" ", "").replace("\n", ...` should do the same thing as this does, and not break compatibility. Sage has only recently (and finally!) became py3 compatible, so keeping py2 compatibility for a while longer makes sense. I think it is easy enough to support both in our case.

**Comment by magv** [comment]
Sorry for keeping this open all this time, and thanks for submitting.
I've committed a solution, and think that python 2 compatibility should now be retired.

**Pull request closed without merging**